config/locales/de.yml

```diff
@@ -599,10 +605,17 @@ de:
   registered_by: Registriert durch
   relatives: Familienangehörige im Haushalt
   remember_me: Angemeldet bleiben
+  reminder_mailer:
+    reminder_email:
+      please_confirm: 'Bitte bestätigen Sie die Stundenrapporte und das Feedback zum Einsatz %{client}.'
+      subject: Rapporte bestätigen
+  reminder_sent: 'Erinnerung wurde versendet an %{email}'
+  reminders: Erinnerungen
   remove_language: Sprache entfernen
   remove_relative: Verwandte entfernen
   reserve: Reservieren
   seeking_clients: Klienten suchen
+  send: Senden
```

Comment:
I'd suggest to use `crud.actions.send: Senden` and in the views with:
```ruby
t_action(:send)
```
Because I doubt that this will be the last use of send in our application 😉